### PR TITLE
Project Templates 3.1.1

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:stop

File: modules/build.gradle [hunk 1]
```diff
@@ -293,4 +293,4 @@ private boolean _hasTestClasses(Project project, String testClassesDirName) {
 	}
 
 	return true
-}
\ No newline at end of file
+}
```
Comment (Ithildir) [inline modules/build.gradle]: Did you add an empty line at the end? Please remove it.

File: modules/sdk/project-templates/project-templates-content-targeting-report/src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -125,4 +126,4 @@
 			</plugin>
 		</plugins>
 	</build>
-</project>
\ No newline at end of file
+</project>
```
Comment (Ithildir) [inline modules/sdk/project-templates/project-templates-content-targeting-report/src/main/resources/archetype-resources/pom.xml]: Did you add an empty line at the end? Please remove it. Also, I think `ProjectTemplateFilesTest` will fail.

File: modules/sdk/project-templates/project-templates-content-targeting-report/src/main/resources/archetype-resources/pom.xml [hunk 3]
```diff
@@ -70,7 +70,8 @@
 				<version>@com.liferay.css.builder.version@</version>
 				<executions>
 					<execution>
-						<phase>generate-sources</phase>
+						<id>default-build-css</id>
```
Comment (Ithildir) [inline modules/sdk/project-templates/project-templates-content-targeting-report/src/main/resources/archetype-resources/pom.xml]: Why do you need it?

**Comment by Ithildir** [comment]
Hi @lawrence-lee! Can you please fix those problems and resend? Thanks! 😊

**Pull request closed without merging**